internal/identity/group/group.go

```diff
@@ -1,5 +1,214 @@
 package group
 
+import (
+	"fmt"
+	"strconv"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+	"github.com/hashicorp/vault/api"
+)
+
 const (
 	LookupPath = "identity/lookup/group"
 )
+
+func GetGroupMember(d *schema.ResourceData, resp *api.Secret, memberField string) (map[string]interface{}, error) {
+	data := map[string]interface{}{}
+
+	switch memberField {
+	case "member_group_ids", "member_entity_ids":
```

Comment:
I see we have constants for these fields, probably want to use them here.

Reply:
Should be fixed in f4f6f03